tox.ini

```diff
@@ -23,6 +23,7 @@ deps=
     dj22:  Django>=2.2,<2.3
     wag23: wagtail>=2.3,<2.4
     wag29: wagtail>=2.9,<2.10
+    wag210: wagtail==2.10rc2
```

Comment:
Change this to `2.10,<2.11` and we're good to go!

Reply:
Done